src/ape_test/provider.py

```diff
@@ -77,7 +77,7 @@ def estimate_gas_cost(self, txn: TransactionAPI, **kwargs) -> int:
             block = self.web3.eth.get_block("latest")
             return block["gasLimit"]
 
-        block_id = kwargs.pop("block_identifier", None)
+        block_id = kwargs.pop("block_identifier", kwargs.pop("block_id", None))
```

Comment:
Here too?

Reply:
if it was `None` before this PR, it doesnt need to change